targets/xbmc

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh -e
```

Comment:
i'm sorry, this went right over my head...  please dumb it down a bit ;)

I believe this is the only remaining item to resolve before this is good to merge.  Thanks a ton for your help! :)
Once merged, i'll reach out to the kodi guys and see if they'll feature it in their blog ;)

I also plan to make a wiki page that talks about it a bit.